src/plans/D3MAaveDaiPlan.sol

```diff
@@ -175,17 +163,14 @@ contract D3MAaveDaiPlan is ID3MPlan {
         uint256 targetInterestRate = bar;
         if (targetInterestRate == 0) return 0;  // De-activated
 
-        uint256 totalDebt = _add(stableDebt.totalSupply(), variableDebt.totalSupply());
+        uint256 totalDebt = stableDebt.totalSupply() + variableDebt.totalSupply();
 
-        uint256 totalPoolSize = _add(
-                TokenLike(dai).balanceOf(adai),
-                totalDebt
-            );
+        uint256 totalPoolSize = TokenLike(dai).balanceOf(adai) + totalDebt;
 
         uint256 targetTotalPoolSize = _calculateTargetSupply(targetInterestRate, totalDebt);
         if (targetTotalPoolSize >= totalPoolSize) {
             // Increase debt (or same)
-            return _add(currentAssets, targetTotalPoolSize - totalPoolSize);
+            return currentAssets + (targetTotalPoolSize - totalPoolSize);
```

Comment:
in theory we could separate the math here to make the subtraction unchecked but doesn't seem to be worth it.